docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -26,7 +26,7 @@ Examples of built-in Rancher extensions are Fleet, Explorer, and Harvester. Exam
 
 :::info
 
-In v2.7.0, the built-in extensions will not be displayed under the **Available** tab. Therefore, you will need to manually add the desired repos to install extensions. We will update the community once these extensions have been pulled out to be available for selection.
+If you've upgraded from an older Rancher version to v2.7.0, uninstalled built-in extensions are no longer listed under the **Available** tab, unless you've manually imported them.
```

Comment:
```suggestion
In v2.7.0, the built-in extensions will not be displayed under the **Available** tab. Therefore, you will need to manually add the desired repos to install extensions.
```

The original text is to indicate a limitation found in certain versions (2.7.0-2.7.?) where users would expect something that's built-in to be available for selection. The update here changes the meaning to indicate that there's a change in existing behavior. Also, extensions were introduced in 2.7 so there is no upgrade scenario involved.